build-scripts/gen-flow-types.ts

```diff
@@ -14,6 +15,18 @@ for (const inFile of files) {
     try {
         execFileSync("yarn", args, {cwd: rootDir});
         console.log(`✅ wrote: ${outFile}`);
+
+        // `flowgen` sometimes outputs code that uses `mixins` instead of `extends`
+        // so we do some post-processing on the files to clean that up.
+        const contents = fs.readFileSync(path.join(rootDir, outFile), "utf-8");
+        if (contents.includes(" mixins ")) {
+            console.log("replacing 'mixins' with 'extends'");
+            fs.writeFileSync(
+                path.join(rootDir, outFile),
+                contents.replace(/ mixins /g, " extends "),
+                "utf-8",
+            );
+        }
```

Comment:
I must've been looking at one of the earlier PRs instead what's on `main` in the wonder-stuff repo.